setup.py

```diff
@@ -2,55 +2,29 @@
 
 import io
 import os
-import re
 
 from setuptools import setup, find_packages
 
-RE_REQUIREMENT = re.compile(r'^\s*-r\s*(?P<filename>.*)$')
-RE_BADGE = re.compile(r'^\[\!\[(?P<text>[^\]]+)\]\[(?P<badge>[^\]]+)\]\]\[(?P<target>[^\]]+)\]$', re.M)
 
-BADGES_TO_KEEP = ['gitter-badge', 'readthedocs-badge']
-
-
-def md(filename):
-    '''
-    Load .md (markdown) file and sanitize it for PyPI.
-    Remove unsupported github tags:
-     - code-block directive
-     - travis ci build badges
```

Comment:
Why is it not relevant anymore ?

Reply:
As you can see on https://regex101.com (ahah) we don't have anything in README.md or CHANGELOG.md that matches that (I got rid of badges for example, and we're not using travis anymore — no idea what code-block is supposed to be)